WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,227 @@
+
+A WEP can be co-authored by more than one community member, but we recommend keeping the number of co-authors small and choosing a representative who will submit the PR to the WEPs repository on behalf of the co-authors.
+
+Draft submission and pre-review
+...............................
+
+A draft WEP must be submitted in a GitHub pull request to the `writethedocs/weps <https://github.com/writethedocs/weps>`_ repository, in the ``drafts/`` directory. If you need help with working with GitHub or creating a PR, please feel free to send an e-mail to support@writethedocs.org, and the Write the Docs team will help you get started.
+
+The WEP draft must follow these guidelines:
+
+- WEP is actionable and implementable. The proposed change must be specific and the resulting action clear.
+- WEP fits the scope of the WEP process, as described in **Scope of WEPs**. WEPs that are out of scope of the process will be rejected by the reviewer.
+- Content format must follow the WEP style as described in the **WEP format and style** section. This includes language, grammar, structure, and markup. The PR might be blocked until proper formatting rules are applied.
+- The WEP content is complete. If you want to submit a Work in Progress (WIP) draft before it is ready for pre-review, you can mark the PR as ``WIP`` and the review team will wait until the draft is ready.
+- **Allow edits from maintainers** option is selected. This helps the reviewer merge your WEP draft if it is accepted.
+- The PR is tagged with a **draft** flag, to indicate that it is in a draft state and is ready for pre-review. 
+
+
+After the pull request for the WEP draft is submitted, a WEP team member reviews the pull request to make sure it adheres to the guidelines. If the WEP is clearly not ready, the reviewer might reject the pull request and ask the author to submit a new WEP after the problems have been fixed.
+
+The reviewer does not vote on the proposed change itself, and only reviews the WEP for scope compliance, format, and completeness. The pre-review period is usually one week, but might change based on the scope of the pre-review feedback and the availability of the author to implement changes.
+
+After the WEP is ready for the community review, the reviewer assigns the **ready-for-discussion** label to the PR. The CoreOps team is then responsible for distributing announcements of each WEP to the relevant community segment. For example, a WEP about meetups will be shared with the Meetup team for distribution between the meetup organizers. For some WEPs, community-wide announcements will be made on Slack or to the mailing list. 
+  
+Community review, discussion, and updates
+.........................................
+
+The standard community review period for a WEP is **30 days**. The WEP is reviewed by a process called **lazy consensus**. This means that community members either provide feedback, approve the WEP explicitly, or ignore the WEP. 
```

Comment:
The "community review" is different from the "reviewer" who looks at scope and compliance? The redundancy in these terms is confusing me

Reply:
Yea, the WEP team does the initial review for acceptance, then the wider community will look at it and have input. There are effectively 2 stages of review.

Reply:
I've changed the language to `decided on`, since reviewed here I agree is ambiguous.